Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2700,6 +2703,42 @@ namespace Babylon
                 std::vector<xr::System::Session::Frame::InputSource::Identifier> squeezeStarts{};
                 std::vector<xr::System::Session::Frame::InputSource::Identifier> squeezeEnds{};
 
+                if (frame.EyeTrackerSpace.first)
+                {
```

Comment:
can you move all of this eye tracking logic to its own ProcessEyeTracking function? The length of ProcessInputSources is making it less and less legible.